config/ganache.json5

```diff
@@ -5,6 +5,7 @@
   },
   storage: {
     enabled: false,
```

Comment:
Well you might want to enable this 😉 Otherwise the address won't be much to use...

Reply:
Sorry guys, I resolve the merge conflicts by mistake. Think it was my PR